using_images/db_images/mongodb.adoc

```diff
@@ -191,35 +222,33 @@ $ oc env dc mongodb \
   MONGODB_ADMIN_PASSWORD=<new_admin_password>
 ----
 
+[IMPORTANT]
+====
+If multiple deployment configurations utilize these environment variables, for
+example in the case of an application created from a template, you must perform
+the above task on each deployment configuration so that the passwords are in
+sync.
+====
```

Comment:
Similarly, if you want to change both passwords, I advise doing it in one shot, otherwise, with a ConfigChange trigger, you'll get useless intermediate deployments.